spec/spec_helper.rb

```diff
@@ -21,8 +21,10 @@
 
 # Requires factories defined in lib/solidus_paypal_commerce_platform/testing_support/factories.rb
 require 'solidus_paypal_commerce_platform/testing_support/factories'
+require 'support/script_tag_helper'
 
 RSpec.configure do |config|
+  config.include ScriptTagHelper
```

Comment:
I would move this inside the `support/script_tag_helper` file and change it to only include the methods when running feature specs.